public/fullSchema.json

```diff
@@ -715,7 +715,7 @@
           "plan_options": {
             "$ref": "#/components/schemas/PlanOptions"
           },
-          "subject_id_settings": {
+          "subject_number_requirements": {
             "$ref": "#/components/schemas/SubjectIdSettings"
```

Comment:
schema file name is not changed, do we want it to be named the same as field?

Reply:
```suggestion
            "$ref": "#/components/schemas/SubjectNumberRequirements"
```

Reply:
fixed it